docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -62,17 +62,13 @@ Full List of Resources:
 12: vlan40	inet 172.17.4.24/24 brd 172.17.4.255 scope global vlan40\   	valid_lft forever preferred_lft forever
 ----
 +
-* vlan30 represents the Storage Network, where the new RGW instances should be
-started on the {CephCluster} nodes.
-* br-ex represents the External Network, which is where in the current
-environment, haproxy has the frontend Virtual IP (VIP) assigned.
+<1> `br-ex` represents the External Network, where in the current
+environment, HAProxy has the front-end Virtual IP (VIP) assigned.
+<2> `vlan30` represents the Storage Network, where the new RGW instances should be started on the {CephCluster} nodes.
```

Comment:
I guess the names of these networks are likely different to customer's environment?